### PR TITLE
Adds Tranzport for Mac 1.0-beta2 to Casks

### PR DESCRIPTION
**Pull request by larsschwegmann**
Hi!
This fork adds https://github.com/steverab/tranzport-mac to the casks.
Thanks for maintaining the casks project, it's really helpful when setting up a new computer!

Thanks in advance! :+1:

### CODE REVIEW

File: Casks/tranzport.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'tranzport' do
+  version '1.0-beta2'
+  sha256 'a03760c312c583d03aa3b57d7c2265eb2c67f806f2cae59b9a6f368d1dcba034'
+
+  url "https://github.com/steverab/tranzport-mac/releases/download/#{version}/Tranzport.app.zip"
```
Comment (vitorgalvao) [inline Casks/tranzport.rb]: Needs a `# github.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by larsschwegmann** [comment]
Sure, just give me a few mins :)

**Comment by vitorgalvao** [comment]
Thank you. Please @mention me after you do, so I get an email to check back.

**Comment by larsschwegmann** [comment]
@vitorgalvao I added the comment and squashed my commits. Please review :)

**Comment by larsschwegmann** [comment]
@vitorgalvao Alright, it should be fine now. The editor I used auto indented line 4 and thus the rubocop was throwing an error. Hope it's ok now :+1:

**Pull request merged**